apps/portal-api/src/modules/services/service-instance.domain.ts

```diff
@@ -133,7 +133,7 @@ export const loadPublicServiceInstances = (context: PortalContext, opts) => {
 
 export const getIsSubscribed = async (context, id) => {
```

Comment:
You can take the opportunity to improve this function naming and parameters typing if you wish 😁 
```suggestion
export const loadIsServiceInstanceSubscribed = async (context: PortalContext, id: ServiceInstanceId) => {
```